resources/views/detailEvenement.blade.php

```diff
@@ -48,7 +48,7 @@
             </div>
             <div id="detailsImages" class=" d-lg-flex">
                 @foreach($groups as $group)
-                    <img src="{{$group->imageurl}}" class="shadow border-0" id="sponsor_img" alt="Evenement afbeelding">
+                    <img src="{{asset('img/'.$group->imageurl)}}" class="shadow border-0" id="sponsor_img" alt="Evenement afbeelding">
```

Comment:
De src is nu "img/https:://..." en de foto's werken nog steeds niet

Reply:
Wat gek, bij mij wel